working/augmentation-libraries/feature-specification.md

```diff
@@ -941,6 +957,11 @@ looks and behaves much like one, and it cannot be augmented as such. It is a
 compile time error to have any augmenting declaration with the same name as the
 representation object.
 
+It is a compile time error if:
+
+*   An extension type augmentation contains a parenthesized clause after the type
+    name.
```

Comment:
Technically, it could be after the extension type's type parameter list too. Maybe:

"contains a representation field clause".